tests/etc/responses.yml.example

```diff
@@ -396,3 +396,12 @@ audiocode_shell:
   save_config: 'Configuration has been saved'
   cmd_response_init: "Test_String1"
   cmd_response_final: "Test_String2"
+
+maipu:
+  base_prompt: maipu_sw
+  router_prompt : maipu_sw>
+  enable_prompt: maipu_sw#
+  interface_ip: 11.11.11.11
+  version_banner: "MyPower (R) Operating System Software"
```

Comment:
Do you know which versions of their OS this driver should work on?

Reply:
This driver can work on  most of theit OS. I will change the desction as Version Information.